ruler-e2e-tests/src/test/kotlin/com/spotify/ruler/e2e/ReleaseReportTest.kt

```diff
@@ -47,6 +47,7 @@ class ReleaseReportTest {
             FileMatcher("/res/layout/activity_main.xml", FileType.RESOURCE, "main-team"),
             FileMatcher("/AndroidManifest.xml", FileType.OTHER, "default-team"),
             FileMatcher("/resources.arsc", FileType.OTHER, "default-team"),
+            FileMatcher("/META-INF/com/android/build/gradle/app-metadata.properties", FileType.OTHER, "default-team"),
```

Comment:
Just for future reference, let's remove this and instead exclude this file once https://issuetracker.google.com/issues/195737366 is fixed